apps/README.md

```diff
@@ -0,0 +1,26 @@
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
+
+## Launch `FluentUI Tester` app on iOS
+
+Prereq: FluentUI Tester on iOS can only run on a Mac.
+
+1. Make sure you have followed the Getting Started instructions [here](../../README.md) to install packages and build the entire Fluent UI React Native repository. E.g. from the root of the repo:
+```
+    yarn && yarn build
+```
+
+2. Then go into `apps/ios/src` folder and run pod install:
+
+```
+    cd apps/ios/src
+    pod install
+```
+Note: if you get the error: "CocoaPods could not find compatible versions for pod "MicrosoftFluentUI"," you will need to run `pod repo update` and then run `pod install` again.
+
+3. Return to the ios directory and run yarn ios to launch the FluentUI Tester app:
+
+```
+    cd ..
+    yarn ios
```

Comment:
Could also do 
```
cd apps/ios
pod install --project-directory=src/
yarn ios
```
to avoid the extra commands. It's technically more keystrokes, but is the more correct command I feel

Reply:
Both are definitely good to me...  I do like the simplicity of what EJ has though

Reply:
Doing it the latter way seems to let cocoapods pick up some extra configs from react-native.config or whatever. 
Both work now, but if we add more custom stuff we miiight have to do it the latter more keystrokes way.

Regardless, for now both work so I'm ok with either. It'd be ok to leave as is

Reply:
Left as is for now, but open to future feedback.